### DISCUSSION TITLE
Lack of terms for viral components?

### DISCUSSION THREAD

**Discussion by uermel** [geneontology/helpdesk#495] [Q&A] [answered]
Hi GO community, I'm a curator/maintainer at the [CZ cryoET data portal](https://cryoetdataportal.czscience.com/). We use GO cellular component terms to assign meaning to image annotations for cryo-electron tomograms. 

We have some data that includes labels for spike proteins of SARS-CoV2, but I'm having a hard time finding a general (or specific) term to associate with the SARS-CoV2 spike protein. I have found the [virion component](https://www.ebi.ac.uk/QuickGO/term/GO:0044423) term and its children, but they don't seem very expressive. 

I'm wondering if there is a reason for the lack of terms regarding viruses in GO? Is this out of scope or just a matter of requesting additional, more specific terms?

**Comment by pgaudet** [comment, marked as answer]
Hi @uermel 
Thanks for contacting GO. My understanding is that the spike protein is a gene product, not a complex or a cellular component. 

In GO we would annotate the protein as being part of the virion membrane - see GO annotations at UniProt: https://www.uniprot.org/uniprotkb/P0DTC2/entry

What other more precise terms were you looking for? 

Thanks, Pascale

**Reply by uermel** [reply]
Hi Pascale, 

this clarifies things. I think I was expecting a slightly more specific term like "viral surface protein". In this case I think it makes sense to stick to established annotations. 

Thanks!
